blueman/bluez/Base.py

```diff
@@ -47,7 +49,7 @@ class Base(Gio.DBusProxy, metaclass=BaseMeta):
     __name = 'org.bluez'
     __bus_type = Gio.BusType.SYSTEM
 
-    __gsignals__ = {
+    __gsignals__: Dict[str, Tuple[Any, GObject.SignalFlags, Tuple]] = {
```

Comment:
The first Tuple starts with GObject.SignalFlags not Any, next is None and then a tuple. For the second tuple we know the first is a str and we know what types to expect from dbus properties which we can represent as a Union. So I think it should be the below which works for Base.
```Dict[str, Tuple[GObject.SignalFlags, None, Tuple[str, Union[str, int, bool, List[str], Dict[str, GLib.Variant], Dict[int, GLib.Variant]], str]]]]```.

But not for subclasses that add their own signals so they will need their own type hints.